specs/eip4844/polynomial-commitments.md

```diff
@@ -449,21 +495,24 @@ def compute_aggregate_kzg_proof(blobs: Sequence[Blob]) -> KZGProof:
 
 ```python
 def verify_aggregate_kzg_proof(blobs: Sequence[Blob],
-                               expected_kzg_commitments: Sequence[KZGCommitment],
-                               kzg_aggregated_proof: KZGProof) -> bool:
+                               expected_kzg_commitments_bytes: Sequence[Bytes48],
+                               kzg_aggregated_proof_bytes: Bytes48) -> bool:
```

Comment:
So these variable names are inconsistent with the other functions. Could we rename these to `expected_commitments` & `aggregated_proof`, respectively. Or if you want to keep the "_bytes" suffix, `expected_commitments_bytes` & `aggregated_proof_bytes`, then add "_bytes" to the other function parameters.